CTags.tmbundle/Support/bootstrap.php

```diff
@@ -3,14 +3,17 @@
 define('DS', DIRECTORY_SEPARATOR);
 define('BUNDLE_SUPPORT', __DIR__ . DS);
 
-if (! isset($_SERVER['CT_PROJECT_ROOT'])) {
-	throw new Exception(
-		'You must set the \'CT_PROJECT_ROOT\' '
-		. 'environment variable in your Project.');
+if (isset($_SERVER['CT_PROJECT_ROOT'])) {
+  define('CT_PROJECT_ROOT', rtrim($_SERVER['CT_PROJECT_ROOT'], DS) . DS);
```

Comment:
Looks like you've changed this file from tab indentation to 2-space? Maintaining the existing style would be nice.